corehq/apps/hqwebapp/templates/hqwebapp/partials/ko_search_box.html

```diff
@@ -1,6 +1,6 @@
 <script type="text/html" id="ko-search-box-template">
     <div class="ko-search-box">
-        <form class="input-group">
+        <form class="input-group" data-bind="submit: clickAction">
```

Comment:
Why isn't this covered by the keypress handler on the input plus the click handler on the search button?

Reply:
This is when the form is submitted (i.e. you hit <enter>). I think that's a different event. This was recently changed to be a `form` from a `div`, which I don't think has that action.